src/pages/ItemsPage.css

```diff
@@ -0,0 +1,24 @@
+main {
+  width: 1200px;
+  margin: 94px auto 0;
+}
+
+.section-title {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.4;
+  letter-spacing: 0.02em;
```

Comment:
관리하기가 어렵다는 점에서 em 사용하시는건 추천드리지 않습니다. 대신 rem을 추천드립니다.

Reply:
덕분에 em, rem에 대해 다시 알아봤습니다! 부모나 다른 조상 모두 letter-spacing이 따로 지정되어 있지 않아 그냥 em->rem으로 변경만 했습니다.

[design(ItemsPage-title): Use rem instead of em](https://github.com/un0211/6-Sprint-Mission/commit/63728f0d0c203207a6e894ebc20306c222947186)